### ISSUE TITLE
Linear regression line using gradient descent is moving in weird ways.

### ISSUE THREAD

**Issue by sotostzam** [sotostzam/artificial-intelligence#12]
**What is the current behaviour?**
Even though the line is initialized at slope 0 and y-intercept 0, after the first iteration is immidiatelly at the center, and moving slowly to as it seems random direction.

**What is the expected behaviour?**
The line should start from the bottom of the plot, and make its way to the best fitting line with the current dataset. This also must take into account the learning rate, which at the time being is equal to 0.01.
